akka-remote/src/main/scala/akka/remote/RemoteActorRefProvider.scala

```diff
@@ -150,7 +148,7 @@ private[akka] class RemoteActorRefProvider(
   // This actor ensures the ordering of shutdown between remoteDaemon and the transport
   @volatile private var remotingTerminator: ActorRef = _
 
-  @volatile private var remoteWatcher: ActorRef = _
+  @volatile private[akka] var remoteWatcher: ActorRef = _
```

Comment:
I would prefer to have this `private _remoteWatcher` and add `private[akka] def remoteWatcher` so that it is clear that it is only set from here.